staking/tests/recouver_account.ts

```diff
@@ -0,0 +1,261 @@
+import {
```

Comment:
Typo in the name of this file : `recouver` -> `recover`

Reply:
Fixed